jekyll/_cci2/jobs-steps.md

```diff
@@ -30,6 +34,26 @@ See the [Specifying Container Images]({{ site.baseurl }}/2.0/executor-types/) do
 
 Steps are a collection of executable commands which are run during a job, the `checkout:` key is required to checkout your code and a key for `run:` enables addition of arbitrary, multi-line shell command scripting.  In addition to the `run:` key, keys for `save_cache:`, `restore_cache:`,  `deploy:`, `store_artifacts:`, `store_test_results:` and `add_ssh_keys` are nested under Steps.
 
+## Sample Configuration with Imported Orb
+
+ ```yaml
+ version: 2.1
+ 
+ orbs:
+   s3tools: circleci/s3@1 #imports the s3 orb in the circleci namespace
```

Comment:
Before we merge this we should get this orb actually live or change it. Let's talk before this merges.

Reply:
is there an update on this @ndintenfass ?

Reply:
We have an unpublished orb called `aws-s3` that I can publish now.
Would you be ok updating this reference to `circleci/aws-s3@1.0.0`?

The orb can be found [here](https://github.com/CircleCI-Public/circleci-orbs/blob/master/src/aws-s3/orb.yml)

The command also becomes just `sync` instead of `s3sync`

Reply:
@ndintenfass, @eddiewebb already has a [PR](https://github.com/CircleCI-Public/circleci-orbs/pull/55) up that is ready to merge and promote this to 1.0.0

Here is the example usage: 
```
version: 2.1
      orbs:
        aws-s3: circleci/aws-s3@volatile
      jobs:
        build:
          docker:
            - image: circleci/python:2.7
          steps:
            - checkout
            - run: mkdir bucket && echo "lorum ipsum" > bucket/build_asset.txt
            - aws-s3/sync:
                from: bucket
                to: "s3://my-s3-bucket-name/prefix"
                overwrite: true
            - aws-s3/copy:
                from: bucket/build_asset.txt
                to: "s3://my-s3-bucket-name"
                arguments: --dryrun
```